### PR TITLE
chore(llc,core,ui,persistence,localizations): 3.4.0

### PR DESCRIPTION
**Pull request by imtoori**

### CODE REVIEW

File: packages/stream_chat/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
...
+- Add check for invalid image URLs
+- Fix `channelState.pinnedMessagesStream` getting reset to `0` after a channel update.
+
+🔄 Changed
+
+- `client.location` is now deprecated in favor of the
+  new [edge server](https://getstream.io/blog/chat-edge-infrastructure) and will be removed in v4.0.0.
+- `channel.banUser`, `channel.unbanUser` is now deprecated in favor of the new `channel.banMember`
+  and `channel.unbanMember` and will be removed in v4.0.0.
+
```
Comment (HayesGordon) [inline packages/stream_chat/CHANGELOG.md:19]: - Added `banExpires` property of type `DateTime` on the `Member`, `OwnUser`, and `User` models.

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
+🐞 Fixed
+
+- SVG rendering fixes.
+- Use file extension instead of mimeType for downloading files.
+- [[#860]](https://github.com/GetStream/stream-chat-flutter/issues/860) CastError while compressing Videos.
+
+✅ Added
+
+- Videos can now be auto-played in `FullScreenMedia`
```
Comment (HayesGordon) [inline packages/stream_chat_flutter/CHANGELOG.md:11]:
```suggestion
- Videos can now be auto-played in `FullScreenMedia`, by setting the `autoplayVideos` argument to true.
```

File: packages/stream_chat_flutter/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
+## 3.4.0
+
```
Comment (HayesGordon) [inline packages/stream_chat_flutter/CHANGELOG.md:2]:
```suggestion
- Updated `stream_chat_flutter_core` dependency to [`3.4.0`](https://pub.dev/packages/stream_chat_flutter_core/changelog).

```

File: packages/stream_chat_flutter_core/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## 3.4.0
+
```
Comment (HayesGordon) [inline packages/stream_chat_flutter_core/CHANGELOG.md:2]:
```suggestion
- Updated `stream_chat` dependency to [`3.4.0`](https://pub.dev/packages/stream_chat/changelog).

```

**Review by HayesGordon — CHANGES_REQUESTED** [review]

File: packages/stream_chat/CHANGELOG.md (hunk 1 above)
Comment (imtoori) [reply packages/stream_chat/CHANGELOG.md:19]:
```suggestion
- Added banExpires property of type DateTime on the Member, OwnUser, and User models.
```

File: packages/stream_chat/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,28 @@
...
+✅ Added
+
+- Added `client.enrichUrl` endpoint for enriching URLs with metadata.
+- Fixed `unreadCount` after removing user from a channel.
```
[the remark this answers is not in the document]
Comment (imtoori) [reply packages/stream_chat/CHANGELOG.md:23]: yeah

File: packages/stream_chat/lib/src/core/models/user.dart [hunk 6]
```diff
@@ -67,6 +68,8 @@ class User extends Equatable {
     'last_active',
     'online',
     'banned',
+    'ban_expires',
+    'dashboard_ban_channel_cid',
```
Comment (xsahil03x) [inline packages/stream_chat/lib/src/core/models/user.dart:72]: maybe we can remove this key as we do not have a field for the same yet.

File: packages/stream_chat_persistence/pubspec.yaml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 name: stream_chat_persistence
 homepage: https://github.com/GetStream/stream-chat-flutter
 description: Official Stream Chat Persistence library. Build your own chat experience using Dart and Flutter.
-version: 3.0.0
+version: 3.0.1
```
Comment (xsahil03x) [inline packages/stream_chat_persistence/pubspec.yaml:4]: I would say we use `3.1.0` instead.

Comment (xsahil03x) [reply packages/stream_chat_persistence/pubspec.yaml:4]:
```suggestion
version: 3.1.0
```

File: packages/stream_chat_persistence/CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,7 @@
+## 3.0.1
```
Comment (xsahil03x) [inline packages/stream_chat_persistence/CHANGELOG.md:1]:
```suggestion
## 3.1.0
```

**Review by xsahil03x — APPROVED** [review]
LGTM

**Review by HayesGordon — DISMISSED** [review]

**Pull request merged**